lib/elixir/lib/string_io.ex

```diff
@@ -23,7 +23,58 @@ defmodule StringIO do
 
   If the `:capture_prompt` option is set to `true`,
   prompts (specified as arguments to `IO.get*` functions)
-  are captured.
+  are captured in the output.
+
+  The device will be created and sent to the function given.
+  When the function returns, the device will be closed. The final
+  result will be a tuple with `:ok` and the result of the function.
+
+  ## Examples
+      iex> {:ok, _contents} = StringIO.open("foo", [], fn(pid) ->
+      ...>     input = IO.gets(pid, ">")
+      ...>     IO.write(pid, "The input was \#{input}")
+      ...>     StringIO.contents(pid)
```

Comment:
Everything here and the example below is indented with one extra space. :)

Reply:
Oh noes! 🤪